client/src/entrypoints/admin/preview-panel.js

```diff
@@ -223,7 +229,7 @@ function initPreview() {
   }
 
   if (WAGTAIL_CONFIG.WAGTAIL_AUTO_UPDATE_PREVIEW) {
-    let oldPayload = new URLSearchParams(new FormData(form)).toString();
+    let oldPayload = '';
```

Comment:
This fixes Bug 3.

We're setting this initial payload to an empty string so that the first `checkAndUpdatePreview()` call will always result in an update.